cpp/include/Ice/Certificate.h

```diff
@@ -18,58 +18,6 @@
 
 namespace IceSSL
 {
-    /**
```

Comment:
Do you plan to remove more from this header in a follow-up PR?

Reply:
Yes, it is still not clear if we should keep it. I would think more about it an open an issue to discuss what we should do.